packages/dapp/src/components/Widgets/Widgets.tsx

```diff
@@ -84,6 +84,8 @@ export function Widgets() {
   }, [activeTab, setActiveTab, starterVariant, starterVariantUsed]);
 
   const handleGetStarted: MouseEventHandler<HTMLDivElement> = (event) => {
+    if (welcomeScreenEntered) return;
```

Comment:
Can we rename this property? it is confunsing. i would read it as he entered IN the welcome screen (he opened it)

Reply:
welcome screen closed

Reply:
Note: Change "welcomeScreenEntered" --> use welcomeScreenClosed